backend/src/main/kotlin/org/loculus/backend/service/groupmanagement/GroupManagementDatabaseService.kt

```diff
@@ -172,4 +152,31 @@ class GroupManagementDatabaseService(
                 contactEmail = it.contactEmail,
             )
         }
+
+    private fun GroupEntity.updateWith(group: NewGroup) {
```

Comment:
Why not move those functions to the file where `GroupEntity` is defined? Extension functions are nice, but in general I think it's better to have all implementations in one place. I can imagine that at least `toGroup` might also be useful in other places.

Reply:
I wasn't sure whether it would be okay to reference the `NewGroup` object from the DB class since it's an API object. I thought maybe those should be kept separate. But I'm happy to move the functions to the object directly.

Reply:
Ah good point. Maybe it's actually better to keep it separated :+1: